From c370f85f31365a02eb4bb6cdf70bba08c7794bea Mon Sep 17 00:00:00 2001
From: Fancy Fang <B47543@freescale.com>
Date: Thu, 26 Dec 2013 16:02:24 +0800
Subject: [PATCH 249/259] ENGR00293365 PXP: change pxp library to match
 pxp_mem_desc struct modification

Since the cpu_addr field in struct pxp_mem_desc has been removed
in linux kernel commit. So in pxp library, all the access to this
field should be removed too.

Signed-off-by: Fancy Fang <B47543@freescale.com>
---
 pxp/pxp_lib.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/pxp/pxp_lib.c b/pxp/pxp_lib.c
index 7b880f5..e1bf2e6 100644
--- a/pxp/pxp_lib.c
+++ b/pxp/pxp_lib.c
@@ -1,7 +1,7 @@
 /*
  * pxp_lib - a user space library for PxP
  *
- * Copyright (C) 2013 Freescale Semiconductor, Inc.
+ * Copyright (C) 2014 Freescale Semiconductor, Inc.
  */
 
 /*
@@ -143,7 +143,6 @@ int pxp_get_phymem(struct pxp_mem_desc *mem)
 {
 	if (ioctl(fd, PXP_IOC_GET_PHYMEM, mem) < 0) {
 		mem->phys_addr = 0;
-		mem->cpu_addr = 0;
 		dbg(DBG_ERR, "PXP_IOC_GET_PHYMEM err\n");
 		return -1;
 	}
@@ -156,7 +155,6 @@ int pxp_put_phymem(struct pxp_mem_desc *mem)
 {
 	if (ioctl(fd, PXP_IOC_PUT_PHYMEM, mem) < 0) {
 		mem->phys_addr = 0;
-		mem->cpu_addr = 0;
 		dbg(DBG_ERR, "PXP_IOC_PUT_PHYMEM err\n");
 		return -1;
 	}
-- 
1.8.0

